tests/tests.py

```diff
@@ -2031,12 +2031,6 @@ def test_plantcv_fill():
     pcv.params.debug_outdir = cache_dir
```

Comment:
These lines can be removed as well since no files are now created by the test function:

```python
    # Test cache directory
    cache_dir = os.path.join(TEST_TMPDIR, "test_plantcv_fill")
    os.mkdir(cache_dir)
    pcv.params.debug_outdir = cache_dir
```